optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -630,10 +629,11 @@ However, this class is also the output of the solution:
 
 === The value range providers
 
-That `timeslotList` field is a value range provider.
+The `timeslotList` field is a value range provider.
 It holds the `Timeslot` instances which OptaPlanner can pick from to assign to the `timeslot` field of `Lesson` instances.
 The `timeslotList` field has an `@ValueRangeProvider` annotation to connect those two,
-by matching the `id` with the `valueRangeProviderRefs` of the `@PlanningVariable` in the `Lesson`.
+by matching the value of `id` field with the `valueRangeProviderRefs` property of the `@PlanningVariable` annotation in the `Lesson` class.
```

Comment:
In the annotation usage `@Foo(bar = "g")` is `bar` a field?

... `id` field with the `valueRangeProviderRefs` property ...
Both `id` and `valueRangeProviderRefs` are the same type. Field or property?

Reply:
Property I'd say.